apps/studio/src/mixins/data_converter.js

```diff
@@ -1,4 +1,7 @@
 import _ from 'lodash'
+import knexlib from 'knex'
+
+const knex = knexlib({ client: 'pg' })
```

Comment:
Hey! This looks to be postgres-specific, but this mixin is used for all the different databases, so this isn't going to work